src/status_im2/common/standard_auth/keycard/view.cljs

```diff
@@ -0,0 +1,165 @@
+(ns status-im2.common.standard-auth.keycard.view
+  (:require [react-native.core :as rn]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.keycard.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [quo2.components.keycard.view :as keycard]
+            [utils.re-frame :as rf]))
+
+(defn handle-number-entry
```

Comment:
Consider making vars private if they don't need to be used outside their own namespaces.

This helps the developer identify what's safer to refactor without causing breakage and without having to rely too much on language servers or the REPL. It also helps developers get more meaningful autocomplete suggestions in `status-mobile`.